composer.json

```diff
@@ -29,7 +29,12 @@
         "illuminate/view": "~5.5.0"
     },
     "require-dev": {
-        "amphp/parallel-functions": "^0.1.0",
+        "amphp/amp": "^2.0.4",
+        "amphp/byte-stream": "^1.2.1",
+        "amphp/parallel": "^0.2.0",
+        "amphp/parallel-functions": "^0.1.1",
+        "amphp/parser": "^1.0.0",
+        "amphp/process": "^0.3.1",
```

Comment:
Why that?

Reply:
You just need `"amphp/amp": "^2.0.3"` here, as that fixed the predictable order of combinators such as `Amp\Promise\all()`.

Reply:
Please depend on `amphp/parallel-functions@^0.1.2` instead.

Reply:
This was just a first stab at getting it to work with the lower recommendations. 

I've changed this, now it follows your recommendations. 

Thanks!